src/backend/vulkan/Cargo.toml

```diff
@@ -25,7 +25,7 @@ byteorder = "1"
 log = { version = "0.4" }
 lazy_static = "1"
 shared_library = { version = "0.1.9", optional = true }
-ash = "0.30"
+ash = "0.31"
 hal = { path = "../../hal", version = "0.5.2", package = "gfx-hal" }
```

Comment:
sorry, one last thing - the `hal` requirement for affected backends needs to be updated

Reply:
Whoops, bad merge resolution